pyth_observer/events.py

```diff
@@ -555,3 +558,95 @@ def get_event_details(self) -> Tuple[str, List[str]]:
             f"CoinGecko Price Chart: {url}",
         ]
         return title, details
+
+
+class PriceStoppedUpdatingCrosschain(PriceAccountValidationEvent):
+    """
+    This alert fires when a cross-chain price feed stopped updating
+    for the past 1 hour.
+    """
+
+    error_code: str = "price-stopped-updating-cross-chain"
+    threshold = 3600  # 3600 seconds = 1 hour
```

Comment:
we need to remember to update this later right?

Reply:
we can always update it to a higher frequency -- chose an arbitrary number of 1 hour for now